node/service/src/chain_spec.rs

```diff
@@ -115,16 +114,6 @@ pub fn get_authority_keys(
 	)
 }
 
-fn testnet_accounts() -> Vec<AccountId> {
-	vec![
-		get_account_id_from_seed::<sr25519::Public>("Alice"),
-		get_account_id_from_seed::<sr25519::Public>("Bob"),
-		get_account_id_from_seed::<sr25519::Public>("Charlie"),
-		get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-		get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-	]
-}
-
 fn member_accounts() -> Vec<AccountId> {
 	vec![
 		(get_account_id_from_seed::<sr25519::Public>("Alice")),
```

Comment:
QUESTION: Are we only keeping `Alice` as default member?

Reply:
Alice, Bob & Charlie. Removed the stash accounts